esp/bindings/SOAP/xpp/sxt/XmlTokenizerException.h

```diff
@@ -31,7 +31,7 @@ namespace sxt {
   class XmlTokenizerException  : public exception{
   public:
     XmlTokenizerException(string exMessage) throw() 
-      : message(exMessage) {}
+      : message(exMessage), row(-1), column(-1) {}
```

Comment:
Even though most of the constructors don't rely on defaults, I think it's still better to initialize default values inline.

It's a better defensive practice in case further constructors are added,.. plus I think you missed a constructor below which kind of demonstrates the point.

Reply:
I guess I can't comment on an unchanged line.  I think you missed the constructor XmlTokenizerException(const XmlTokenizerException& other) below which doesn't init those values.